server/room.mjs

```diff
@@ -643,6 +643,11 @@ export default class Room {
     this.broadcast('mouse', { player: player.name, mouseState });
   }
 
+  moveFile(source, target) {
+    fs.copyFileSync(source, target);
```

Comment:
Could use `fs.copyFilesSync(source, target, fs.constants.COPYFILE_FICLONE)` to copy via reflink if possible falling back on normal copy, on supported filesystems this should make this perform similarly to rename.